acceptance/lib/puppet_x/acceptance/external_cert_fixtures.rb

```diff
@@ -1,361 +0,0 @@
-module PuppetX
-module Acceptance
-class ExternalCertFixtures
```

Comment:
This is needed by puppetserver when running `server` tests, so can't be deleted.

Reply:
Actually it's not needed. Puppetserver has its own [copy of the file](https://github.com/puppetlabs/puppetserver/blob/master/acceptance/suites/tests/certificate_authority/split_external_cas.rb) and the external CAs test passes:

```
 env PUPPET_BUILD_VERSION=c37313cd3429916b3e656014869f8a239fb70714 bundle exec beaker --debug --root-keys --preserve-hosts onfail --type aio --helper acceptance/lib/helper.rb --options-file acceptance/config/beaker/options.rb --config ./redhat7-64m-64.cfg --pre-suite acceptance/suites/pre_suite/foss/ --tests acceptance/suites/tests/certificate_authority/split_external_cas.rb --load-path acceptance/lib,ruby/puppet/acceptance/lib/
```